commit 4a8da7e02dc1b882d33cfbb7301d29bdb5ea915f
Author: Miroslav Lichvar <mlichvar@redhat.com>
Date:   Wed Jun 11 14:53:47 2025 +0200

    examples: improve chrony.conf examples
    
    Add a note that three servers is the generally recommended minimum for
    an NTP client to be able to detect a falseticker. Mention that the pool
    directive uses four servers. Update the links to the pool join page and
    list of public servers.

diff --git a/examples/chrony.conf.example1 b/examples/chrony.conf.example1
index 5e93ea75..f822813b 100644
--- a/examples/chrony.conf.example1
+++ b/examples/chrony.conf.example1
@@ -1,4 +1,4 @@
-# Use public NTP servers from the pool.ntp.org project.
+# Use four public NTP servers from the pool.ntp.org project.
 pool pool.ntp.org iburst
 
 # Record the rate at which the system clock gains/losses time.
diff --git a/examples/chrony.conf.example2 b/examples/chrony.conf.example2
index 03e7d47b..a257f54c 100644
--- a/examples/chrony.conf.example2
+++ b/examples/chrony.conf.example2
@@ -1,5 +1,10 @@
-# Use public servers from the pool.ntp.org project.
-# Please consider joining the pool (https://www.pool.ntp.org/join.html).
+# Note: The general recommendation for an NTP client is to have at least
+# three NTP servers to be able to detect one server providing incorrect
+# time (falseticker).
+
+# Use four public NTP servers from the pool.ntp.org project. If this
+# host has a static public IP address, please consider joining the pool:
+# https://www.ntppool.org/join.html
 pool pool.ntp.org iburst
 
 # Record the rate at which the system clock gains/losses time.
diff --git a/examples/chrony.conf.example3 b/examples/chrony.conf.example3
index 8d895d04..01eaff1c 100644
--- a/examples/chrony.conf.example3
+++ b/examples/chrony.conf.example3
@@ -21,10 +21,12 @@
 #######################################################################
 ### SPECIFY YOUR NTP SERVERS
 # Most computers using chrony will send measurement requests to one or
-# more 'NTP servers'.  You will probably find that your Internet Service
+# more NTP servers.  The general recommendation is to have at least
+# three NTP servers to be able to detect one server providing incorrect
+# time (falseticker).  You will probably find that your Internet Service
 # Provider or company have one or more NTP servers that you can specify.
 # Failing that, there are a lot of public NTP servers.  There is a list
-# you can access at http://support.ntp.org/bin/view/Servers/WebHome or
+# you can access at https://support.ntp.org/bin/view/Servers/WebHome or
 # you can use servers from the pool.ntp.org project.
 
 ! server ntp1.example.net iburst
